addons/nuxeo-diff/nuxeo-diff-content/src/main/resources/OSGI-INF/content-diff-config.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.diff.settings">
+
+  <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+    <property name="nuxeo.diff.limit">10000</property>
+  </extension>
```

Comment:
Add a description for this property, like it's done for others. Add also a `since` to help devs.